.processes/development.md

```diff
@@ -55,6 +55,13 @@ _Current:_
 - [Tino](https://github.com/tolbrino) until 05/02/2022
 - [Robert](https://github.com/robertkiel) until 04/03/2022
 
+#### Electing a new representantive
+
+- every election should begin 5 working days prior to the representantive's replacement
+- voting is done via google form similar to [this](https://docs.google.com/forms/d/1CA_C_NgI9M_drHFqOOVlBVdfuiWsg64xE_wE1V1pgyw)
```

Comment:
I suggest to keep points like that technology neutral. The Google Form can stay as a reference